src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -1422,7 +1422,7 @@ def uploadCharacteristics(self, charBufferNumber = FINGERPRINT_CHARBUFFER1, char
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
         ## Upload data packets
-        packetNbr = len(characteristicsData) / maxPacketSize
+        packetNbr = int(len(characteristicsData) / maxPacketSize)
```

Comment:
Abbreviation are not great for understanding imho, please rename the variable(s)